ci-operator/config/openshift/cincinnati/openshift-cincinnati-release-4.4.yaml

```
@@ -0,0 +1,133 @@
+base_images:
+  centos:
+    name: centos
+    namespace: ci
+    tag: "7"
+  openshift_release_golang-1.13:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+build_root:
+  image_stream_tag:
+    name: cincinnati-build-root
+    namespace: cincinnati-ci
+    tag: buildroot
+images:
+- dockerfile_path: dist/openshift-release/Dockerfile.builder
+  from: centos
+  to: buildroot
+- dockerfile_path: dist/Dockerfile.deploy/Dockerfile
+  from: centos
+  inputs:
+    buildroot:
+      as:
+      - quay.io/app-sre/cincinnati:builder
+  to: deploy
+- dockerfile_path: dist/Dockerfile.e2e/Dockerfile
+  from: centos
+  inputs:
+    buildroot:
+      as:
+      - quay.io/app-sre/cincinnati:builder
+    openshift_release_golang-1.13:
+      as:
+      - registry.svc.ci.openshift.org/openshift/release:golang-1.13
+  to: e2e-test
+promotion:
+  name: cincinnati-build-root
+  namespace: cincinnati-ci
+resources:
+  '*':
+    requests:
+      cpu: 1000m
+      memory: 1Gi
+  cargo-test:
+    requests:
+      cpu: "3"
+      memory: 5Gi
+tag_specification:
+  name: "4.5"
```

```suggestion
  name: "4.4"
```
Lets run Cincinnati 4.4 on 4.4 cluster

---

Can you elaborate a bit why you suggest that? It honestly seems arbitrary to me as I'm not aware of any specific OCP version requirements.

---

I assume customers would run OSUS 4.4 on 4.4 clusters, as in 4.5 the operator may want to use some more features.

Not sure if this is corrent though

---

The only version coupling I'm aware of is that they would point OCP 4.4 clusters to a OSUS 4.4 instance.